services/dynamic-sidecar/src/simcore_service_dynamic_sidecar/modules/long_running_tasks.py

```diff
@@ -57,6 +58,26 @@ async def _retry_docker_compose_start(
     return await docker_compose_start(compose_spec, settings)
 
 
+@retry(
+    wait=wait_random_exponential(),
```

Comment:
what is max in this case? just be sure it's below 5 minutes... .elsewise it is useless

Reply:
I think that what SAN refers is that the you have to bound the randomness of your ``wait_random_exponential`` function since it might decide e.g. to wait before retry say 5 minutes and therefore it would effectively NOT retry or only do it once

Check https://tenacity.readthedocs.io/en/latest/#waiting-before-retrying

Besides, I think we should adopt a clear criteria about the policies we follow for retry. I find it quite random the way we decide. If you have some ideas, perhaps you can share them and we can write some rules?

Reply:
I now see to what you are referring to. I've decided to bound it at max to a 30 second wait interval.